src/views/Correction/CoopCorrection.vue

```diff
@@ -134,7 +134,7 @@ export default class CoopCorrection extends Mixins(CommonMixin, DateMixin, FeeMi
 
   /** The original filing date, in Pacific time. */
   get originalFilingDate (): string {
-    return this.apiToPacificDateLong(this.getCorrectedFilingDate)
+    return DateUtilities.yyyyMmDdToPacificDate(this.getCorrectedFilingDate, true)
```

Comment:
The Date Mixin is deprecated, so I'm using Date Utilities instead.

And the fix is to properly convert from yyyy-mm-dd.